benchmark_v2/groupby_benchmark.py

```diff
@@ -0,0 +1,50 @@
+import arkouda as ak
+import pytest
+
+TYPES = ["int64", "bigint", "str"]
```

Comment:
I think we sometimes want `"mixed"` as an option which gives a mix of str and int

Reply:
Added `mixed` option. Thanks for catching that. I added it as a defaults as well.